src/app/ngrx/model/metrics.model.ts

```diff
@@ -13,17 +13,11 @@ export type MetricsType = 'downloads' | 'geo' | 'userAgents';
 export interface PodcastMetricsModel {
   seriesId: number;
   feederId: string;
-  episodeIdsNotInOthers?: number[];
```

Comment:
What were these "others" for?

Reply:
This "others" business was a bad idea I had about needing to have the podcasts metrics minus the episodes metrics in the state so that I could just get the 'All Other Episodes' from the state. It was more complicated to manage with incoming CASTLE_PODCAST_METRICS and CASTLE_EPISODE_METRICS and needing to know where we are in the cycle of responses to start subtracting from podcast data. It seems less complicated to let the component decide if it has both podcast and episode data in order to do the subtraction when it needs it rather than as the data comes into state.

Reply:
Makes sense.